Mathlib/FieldTheory/IntermediateField/Basic.lean

```diff
@@ -612,6 +612,16 @@ theorem mem_lift {F : IntermediateField K L} {E : IntermediateField K F} (x : F)
     x.1 ∈ lift E ↔ x ∈ E :=
   Subtype.val_injective.mem_set_image
 
+/--The algEquiv between an intermediate field and its lift-/
+def lift_algEquiv (E : IntermediateField K L) (F : IntermediateField K E) : ↥F ≃ₐ[K] lift F where
+  toFun x := ⟨x.1.1,(mem_lift x.1).mpr x.2⟩
+  invFun x := ⟨⟨x.1, lift_le F x.2⟩, (mem_lift ⟨x.1, lift_le F x.2⟩).mp x.2⟩
+  left_inv := congrFun rfl
+  right_inv := congrFun rfl
+  map_mul' _ _ := rfl
+  map_add' _ _ := rfl
+  commutes' _ := rfl
```

Comment:
I don't this is used anywhere, but if you intend to keep it, could you add some basic API like `lift_algEquiv_apply`?

(Also, maybe it should be called `liftAlgEquiv` since it's a def and not a theorem?)

Reply:
This lemma would be used in the incoming PR #16982 because I got trouble proving the lift of a finite dimensional intermediatefield is also finite dimensional, maybe there are better ways to solve it. But atleast for now I should add `liftAlgEquiv_apply`